### PR TITLE
Add ITS_LIVE dataset

### PR DESCRIPTION
**Pull request by jhkennedy**
This:

* Add the ITS-LIVE data to the repository README
* Adds an `its-live/README.md` describing the data
* A GitHub Action to automatically copy the `its-live/README.md` to  the `its-live-data` bucket on merges to `main`
  * The bucket copy of the README is currently listed as the dataset documentation in the AWS Open Data catalog, see: https://github.com/awslabs/open-data-registry/pull/2071

### CODE REVIEW

File: its-live/README.md
Comment (jhkennedy) [inline its-live/README.md:1]: @alex-s-gardner, @markf6, @mliukis, @betolink we need some documentation of the ITS_LIVE data as it is in S3. I've started a README here, but I'm going to need input on what to include.

File: its-live/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+* `isce_autoRIFT/`: 
+* `month-data-logs/`: 
+* `mosaics/`: 
+* `qgis_project/`: 
+* `rgb_mosaics/`: 
+* `test/`: 
+* `test_datacubes/`: 
+* **`vel_web_tiles/`**: 
+* **`velocity_image_pair/`**: 
+* **`velocity_mosaic/`**:
```
Comment (jhkennedy) [inline its-live/README.md]: @alex-s-gardner, @mliukis, I'd like to explain (broadly) what each prefix is for in the bucket. Are we going to keep all of them? I expect many aren't going to be particularly interesting to users, so I'd like to bold the ones users will care about. 

Alternatively, we can just list the ones users should care about, or break it into two sections.

File: its-live/README.md
Comment (jhkennedy) [reply its-live/README.md:1]: It's probably best to "view file" to review this document. You should also be able to fork and propose changes to this branch.

Comment (jhkennedy) [reply its-live/README.md:1]: Some good examples of this:
* https://scedc.caltech.edu/data/cloud.html
* https://github.com/gbif/occurrence/blob/master/aws-public-data.md
* https://glo-30-hand.s3.us-west-2.amazonaws.com/readme.html

File: its-live/README.md (hunk 1 above)
Comment (mliukis) [reply its-live/README.md]: @jhkennedy The following folders are not end user intended:

- [ ] NSIDC - all corrected v01 data products that we ingested into NSIDC
     @alex-s-gardner should we remove all of them on our side? May be good to keep them as a reference until we are done with v02 ingest into NSIDC?
- [ ] test_datacubes - this is our test data (chunking, verification of new data, etc.)
- [x] catalog_geojson_original - removed
- [x] catalog_geojson_latest - removed
- [ ] catalog_geojson - @alex-s-gardner do we want to expose catalog geojsons to the user?

These look like some staged data for temporary use (?):
- [ ] Test:
    2023-07-24 09:31:04  209641057 N52W175.nc
- [ ] L7_PV_fix:
    2023-07-17 10:18:55  238194861 l7_pvalue_comp_with_fix.parquet
- [ ] month-data-logs: @alex-s-gardner there are some monthly logs from 2022. Do we still need them?
- [ ] test - some test data

Comment (jhkennedy) [reply its-live/README.md]: Going mark this resolved and will track what data is going where in: https://github.com/ASFHyP3/OpenData/issues/10

**Pull request merged**